protos/vega/spec_definition.go

```diff
@@ -2,15 +2,19 @@ package vega
 
 import (
 	datapb "code.vegaprotocol.io/vega/protos/vega/data/v1"
+	"google.golang.org/protobuf/types/known/structpb"
 )
 
+type DataSourceContentType int32
+
 const (
-	DataSourceDefinitionTypeUspec = 0
-	DataSourceDefinitionTypeInt   = 1
-	DataSourceDefinitionTypeExt   = 2
+	DataSurceContentTypeInvalid DataSourceContentType = iota
+	DataSourceContentTypeOracle
+	DataSourceContentTypeEthOracle
+	DataSourceContentTypeInternalTimeTermination
 )
 
-func (d DataSourceDefinition) DeepClone() *DataSourceDefinition {
+func (d DataSourceDefinition) DeepClone() *DataSourceDefinition { // TODO: Test this
```

Comment:
@AnNiran this is needed?

Reply:
Yes, this a todo note left because i have to add the test for this method.

Reply:
```suggestion
func (d DataSourceDefinition) DeepClone() *DataSourceDefinition {
```

Reply:
Suggesting to remove it for now, i'll cover it in my next PR.